### ISSUE TITLE
Incorrect array indentation

### ISSUE THREAD

**Issue by jaredp** [palantir/tslint#8]
tslint incorrectly flags indentation of object literals in arrays such as

``` ts
[
    {
        a: 4
    }, {
        b: 10
    }
]
```
